system/scripts/zone/core/calc_combat.cs

```diff
@@ -132,37 +142,43 @@ public float SCR_CalculateDamage(ICombatEntity attacker, ICombatEntity target, S
 			target.TrySpendSp(maxSp * spRate);
 		}
 
-		var sizeBonusDamage = SCR_SizeTypeBonus(attacker, target, skill, skillHitResult);
+		var sizeBonusDamage = SCR_SizeTypeBonus(attacker, target, skill, modifier, skillHitResult);
 		if (sizeBonusDamage != 0)
 		{
 			damage += sizeBonusDamage;
 		}
 
-		var attrMultiplier = SCR_AttributeMultiplier(attacker, target, skill, skillHitResult);
+		var attrMultiplier = SCR_AttributeMultiplier(attacker, target, skill, modifier, skillHitResult);
 		if (attrMultiplier != 1)
 		{
 			damage *= attrMultiplier;
 		}
 
-		var atkTypeMultiplier = SCR_AttackTypeMultiplier(attacker, target, skill, skillHitResult);
+		var atkTypeMultiplier = SCR_AttackTypeMultiplier(attacker, target, skill, modifier, skillHitResult);
 		if (atkTypeMultiplier != 1)
 		{
 			damage *= atkTypeMultiplier;
 		}
 
-		var raceMultiplier = SCR_RaceMultiplier(attacker, target, skill, skillHitResult);
+		var raceMultiplier = SCR_RaceMultiplier(attacker, target, skill, modifier, skillHitResult);
 		if (raceMultiplier != 1)
 		{
 			damage *= raceMultiplier;
 		}
 
-		var hitCountMultiplier = SCR_HitCountMultiplier(attacker, target, skill, skillHitResult);
+		var hitCountMultiplier = SCR_HitCountMultiplier(attacker, target, skill, modifier, skillHitResult);
 		if (hitCountMultiplier != 1)
 		{
 			damage *= hitCountMultiplier;
 			skillHitResult.HitCount = (int)Math.Round(skillHitResult.HitCount * hitCountMultiplier);
 		}
 
+		if (target.Components.Get<BuffComponent>().TryGet(BuffId.Cloaking_Buff, out var cloakingBuff))
+		{
+			// Cloaking reduces damage by 25%
+			damage = Math.Max(1, damage - damage * 0.75f);
```

Comment:
I'm kind of tired right now, but aren't you reducing the damage by 75% here? Or is that part correct and I'm misunderstanding the comment?

Reply:
Nope, you're right, that does accidentally reduce it by 75%.  Whoops.